features/project_test.feature

```diff
@@ -0,0 +1,286 @@
+@api @test
+Feature: Test
+
+  Users should be able to retrieve the test details by its id.
```

Comment:
Prefer `a test`, or `the details of a test`, or `test details` to `the test details` (which sounds weird).